### PR TITLE
Add support for an 'operations' part when doing a multipart POST.

### PR DESCRIPTION
**Pull request by osi**
This will allow the https://github.com/jaydenseric/graphql-multipart-request-spec
to be supported. [Apollo File Uploads](https://www.apollographql.com/docs/guides/file-uploads.html) are built on top of this.

### CODE REVIEW

File: src/main/java/graphql/servlet/AbstractGraphQLHttpServlet.java [hunk 1]
```diff
@@ -109,7 +109,28 @@ public AbstractGraphQLHttpServlet(List<GraphQLServletListener> listeners, boolea
                                     Collections::singletonList,
                                     (l1, l2) -> Stream.concat(l1.stream(), l2.stream()).collect(Collectors.toList())));
 
-                    if (fileItems.containsKey("graphql")) {
+                    if(fileItems.containsKey("operations")) {
+                        final Optional<Part> queryItem = getFileItem(fileItems, "operations");
+                        if (queryItem.isPresent()) {
```
Comment (oliemansm) [inline src/main/java/graphql/servlet/AbstractGraphQLHttpServlet.java]: We're starting to get a lot of duplication here. This `if` block seems to be an exact duplicate of the one below in the `fileItems.containsKey("graphql")` block. Would you mind refactoring a bit by extracting that block in a separate method and replacing both blocks with a call to that method?

Comment (osi) [reply src/main/java/graphql/servlet/AbstractGraphQLHttpServlet.java]: Yes, no problem. I'm also adding built-in support for the `map` parameter that the multipart request spec uses, as it's not possible to layer in that behavior as a library consumer. I'll include this refactoring in my next update.

**Comment by rsredsq** [comment]
Hello colleagues,

Any updates on this PR?

Thanks

**Comment by osi** [comment]
@rsredsq If you have an immediate need, you can use my branch via jitpack while awaiting merge (that's what i'm doing) ... 
https://jitpack.io/#osi/graphql-java-servlet/6036b71163

**Pull request merged**